README.md

```diff
@@ -399,6 +399,9 @@ Supported customizable components:
  * `TableResize`
  * `TableToolbar`
  * `TableToolbarSelect`
+ * `Tooltip`
+
+For more information, please see this [example](https://github.com/gregnb/mui-datatables/examples/custom-components/index.js). Additionally, all examples can be viewd [live](https://codesandbox.io/s/github/greg/mui-datatables) at our CodeSandbox.
```

Comment:
>https://codesandbox.io/s/github/greg/mui-datatables

Could not find the specified repository or directory

Reply:
Good catch! Thank you for checking that - I've updated the README, please let me know if you see anything else like that. I looked into editing the description and it appears only project owners can edit it (I'm just a maintainer). I may hit greg up later about it but I don't want to bug him too much (just asked him a question earlier today so I'll wait a bit).

Reply:
@patorjk Thanks~